frontend/src/util/CreatePortal.tsx

```diff
@@ -0,0 +1,23 @@
+import { PropsWithChildren, useEffect, useRef, useState } from 'react';
+import ReactDOM from 'react-dom';
+
+interface CreatePortalProps {
+  modalId: 'snack-bar';
+}
+
+const CreatePortal = ({ modalId, children }: PropsWithChildren<CreatePortalProps>) => {
+  const portalRef = useRef<Element | null>(null);
```

Comment:
useRef의 타입으로 Element와 null을 함께 받아주신 이유가 있을까요?

Reply:
최초의 값은 null(document를 못찾는 상태)여서 null로 넣어주었습니다 해당 구문을 넣어주지 않으면 타입에러가 났던 기억이... 정확히 확인해보겠습니다

Reply:
다시 한번 확인해보니 최초의 값이 Null일 수있어서 Null을 넣어야 타입에러가 발생하지 않네요!

Reply:
Ref의 current를 직접 접근하게 되어서 발생하는 타입 에러인 것 같아요.
이렇게 useRef를 사용하지 않으면 코드를 줄일 수 있지 않을까요?

```suggestion
  const CreatePortal = ({ modalId, children }: PropsWithChildren<CreatePortalProps>) => {
  const [mounted, setMounted] = useState(false);

  useEffect(() => {
    setMounted(true);
  }, []);

  const target = document.querySelector<HTMLElement>(`#${modalId}`);
  return mounted && target ? ReactDOM.createPortal(children, target) : <></>;
};
```

Reply:
반영해서 수정했습니다!